config/initializers/assets.rb

```diff
@@ -9,3 +9,10 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+# Mojular template assets
+bowerrc_dir = JSON.parse(IO.read("#{Rails.root.to_s}/.bowerrc"))['directory']
+Rails.application.config.assets.paths << Rails.root.join(bowerrc_dir)
+Rails.application.config.assets.paths << Rails.root.join(bowerrc_dir, 'mojular', 'assets')
+Rails.application.config.assets.paths << Rails.root.join(bowerrc_dir, 'govuk-template', 'assets')
```

Comment:
This might be clearer as:

```ruby
Rails.application.config.assets.paths += [
  Rails.root.join(bowerrc_dir),
  Rails.root.join(bowerrc_dir, 'mojular', 'assets'),
  Rails.root.join(bowerrc_dir, 'govuk-template', 'assets'),
]
```